api/src/main/kotlin/io/spine/protodata/renderer/SourceFile.kt

```diff
@@ -110,10 +119,38 @@ private constructor(
      *
      * Insertion points should be marked with comments of special format. The added code is always
      * inserted after the line with the comment, and the line with the comment is preserved.
+     *
+     * @see atInline
      */
     public fun at(insertionPoint: InsertionPoint): SourceAtPoint =
         SourceAtPoint(this, insertionPoint)
 
+    /**
+     * Creates a new fluent builder for adding code at the given inline [insertionPoint].
+     *
+     * If the [insertionPoint] is not found in the code, no action will be performed as the result.
+     * If there are more than one instances of the same insertion point, the code will be added to
```

Comment:
"instance"

And again, here we speak about multiple instances of the same insertion point. See my comment on this matter above.